config/mgc-install-guide/cert-manager/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+resources:
+  - ../../cert-manager
+
+patchesStrategicMerge:
+  - |-
+    apiVersion: admissionregistration.k8s.io/v1
+    kind: ValidatingWebhookConfiguration
+    metadata:
+      name: mgc-cert-manager-webhook
+    $patch: delete
+  - |-
+    apiVersion: admissionregistration.k8s.io/v1
+    kind: MutatingWebhookConfiguration
+    metadata:
+      name: mgc-cert-manager-webhook
+      namespace: cert-manager
+    $patch: delete
```

Comment:
We have this duplicated in two places (here and quickstart), and local-setup does the equivalent manually. This should be in one place and used everywhere where we installing cert-manager. Considering `config/cert-manager` is our kustomization of cert manager installs and we remove these in all installations I'd be in favour of just adding this to `config/cert-manager/kustomization.yaml` 

@roivaz Do we use this elsewhere for real deployments where we might want webhooks enabled?

@eoinfennessy We can leave it like this for now, I'll look into changing it as part of a config directory cleanup.